Source/Diagnostics/WarpXOpenPMD.cpp

```diff
@@ -371,6 +371,10 @@ namespace detail
                                           {openPMD::UnitDimension::I,  1},
                                           {openPMD::UnitDimension::T,  1},
                                   });
+        } else if (field_name.substr(0,2) == "T_"){ // temperature in eV
+            mesh.setUnitDimension({
+                                          {openPMD::UnitDimension::theta,  1},
```

Comment:
Can I ask the rationale behind defining a special unit for temperature? If openPMD expresses quantities with respect to SI, shouldn't this be eV [energy]/[charge]-> [mass][length]^2[time]^-1[current]^-1?